administrator/language/de-DE/com_users.ini

```
@@ -318,7 +318,7 @@ COM_USERS_TOOLBAR_ACTIVATE="Aktivieren"
 COM_USERS_TOOLBAR_BLOCK="Sperren"
 COM_USERS_TOOLBAR_MAIL_SEND_MAIL="Senden"
 COM_USERS_TOOLBAR_UNBLOCK="Freigeben"
-COM_USERS_UNACTIVATED="Deaktiviert"
+COM_USERS_UNACTIVATED="Noch nicht aktiviert"
```

```suggestion
COM_USERS_UNACTIVATED="Inaktiv"
```

3 Wörter sind doch etwas lang

---

Inaktiv sounds good. He has to be activated and before that he is "inaktiv"